.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
It looks like this file got mistakenly added.

Reply:
Removed.